src/pybind/mgr/cephadm/module.py

```diff
@@ -1646,8 +1646,11 @@ def _create_daemon(self,
             extra_args.extend(['--config-json', '-'])
         else:
             # Ceph.daemons (mon, mgr, mds, osd, etc)
+            partial_auth_key = daemon_id
+            if daemon_type == 'crash': #
+                partial_auth_key = host
```

Comment:
can you move this code into utils.name_to_auth_entity?

Reply:
Done.
Note: It seems that mgr also changed his auth key

Reply:
thanks!